blockchain/txvalidator.go

```diff
@@ -1119,6 +1120,12 @@ func checkTransactionPayload(txn *Transaction) error {
 	case *payload.CoinBase:
 	case *payload.SideChainPow:
 	case *payload.WithdrawFromSideChain:
+		if txn.PayloadVersion == payload.WithdrawFromSideChainVersionV2 {
```

Comment:
need to move the check from sanity to context

Reply:
Fixed it